crates/vm2-interface/src/lib.rs

```diff
@@ -23,9 +23,9 @@
 //! version that you publish and import it from the previous version instead.
 //!
 //! This is how you would add a new method to StateInterface and a new opcode.
+//!
 //! ```
-//! # use eravm_stable_interface as eravm_stable_interface_v1;
```

Comment:
I'd keep this renaming to make clearer that you are supposed to have the older version as a dependency.